envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EndpointMetadataMergingTests.kt

```diff
@@ -10,6 +10,7 @@ import pl.allegro.tech.servicemesh.envoycontrol.config.envoy.EnvoyExtension
 import pl.allegro.tech.servicemesh.envoycontrol.config.envoycontrol.EnvoyControlExtension
 import pl.allegro.tech.servicemesh.envoycontrol.config.service.EchoServiceExtension
 
+// flaky test
```

Comment:
Maybe it is good idea to create another test profile similar to existing Resilience. And group all flaky tests in this profile. Then we can create separate Github Action which will run only flaky tests. 

Thanks to that we will be able easily finds that, CI doesn't pass because of flaky and in addition we will can rerun only this tests which will decrease time needed to wait